lib/workers/repository/error-config.ts

```diff
@@ -26,7 +26,7 @@ export async function raiseConfigWarningIssue(
     logger.debug('Updating onboarding PR with config error notice');
     body = `## Action Required: Fix Renovate Configuration\n\n${body}`;
     body += `\n\nOnce you have resolved this problem (in this onboarding branch), Renovate will return to providing you with a preview of your repository's configuration.`;
-    if (GlobalConfig.get('dryRun')) {
+    if (GlobalConfig.get('dryRun') === 'full') {
```

Comment:
```suggestion
    if (GlobalConfig.get('dryRun')) {
```

We don't want to do this under any dry run scenario